main.go

```
@@ -40,12 +41,15 @@ func init() {
 
 	_ = gatewayv2alpha1.AddToScheme(scheme)
 	_ = networkingv1alpha3.AddToScheme(scheme)
+	_ = rulev1alpha1.AddToScheme(scheme)
 	// +kubebuilder:scaffold:scheme
 }
 
 func main() {
 	var metricsAddr string
 	var enableLeaderElection bool
+	var oathkeeperSvcAddr string
+	flag.StringVar(&oathkeeperSvcAddr, "oathkeeper service address", "ory-oathkeeper-proxy.ory.svc.cluster.local", "Oathkeeper proxy service")
```

This flag should not have any default, but just panic if not provided by user. We shouldn't try to "guess" system configuration.